### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by MohamedShatry**
A few changes in this PR
- Removed icon path in index
- Added a post class to hold incoming messages
- Changed the way querying works
- Update logic for querying messages

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/UsersServlet.java [hunk 1]
```diff
@@ -13,43 +13,30 @@
...
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List;
-import com.google.datastore.v1.PropertyFilter;
+import org.apache.commons.io.IOUtils;
```
Comment (kiprot) [inline src/main/java/com/google/sps/servlets/UsersServlet.java:31]: what does IOUtils do

File: src/main/java/com/google/sps/servlets/UserServlet.java [hunk 2]
```diff
@@ -13,27 +13,23 @@
...
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List;
-
+import org.apache.commons.io.IOUtils;
+import com.google.sps.servlets.User;
 
-FirebaseApp.initializeApp();
 /** Servlet that holds the users on this WebApp */
```
Comment (kiprot) [inline src/main/java/com/google/sps/servlets/UserServlet.java:33]: this comment is similar to the one on UsersServlet.java, line 34. Do they both do the same thing?

File: src/main/java/com/google/sps/servlets/UsersServlet.java (hunk 1 above)
Comment (MohamedShatry) [reply src/main/java/com/google/sps/servlets/UsersServlet.java:31]: It allows us to accept JSON objects as input to the servlet, sent through the request body

File: src/main/java/com/google/sps/servlets/UserServlet.java (hunk 2 above)
Comment (MohamedShatry) [reply src/main/java/com/google/sps/servlets/UserServlet.java:33]: The ioutils? Yes they do

Comment (kiprot) [reply src/main/java/com/google/sps/servlets/UserServlet.java:33]: i was referring to the comment /** Servlet that holds the users on this WebApp */

**Pull request merged**